docpad.coffee

```diff
@@ -111,11 +110,11 @@ module.exports =
       url: 'https://goo.gl/forms/Jyeqyrd1dwi2w3752'
 
     # List of Sponsors
-    sponsors: [
+    # sponsors: [
       # name: 'Eventick'
       # logo: 'themes/yellow-swan/img/sponsor.png'
       # url: 'http://eventick.com.br'
-    ]
+    # ]
```

Comment:
And also this.

Reply:
As a suggestion, you can change the _your logo here_ to a _seja um patrocinador_ text/image.
Note that this is on the layout itself and not on `docpad.coffee`.